### ISSUE TITLE
ROI Selection

### ISSUE THREAD

**Issue by mohits19** [cleebp/csc-522-team5#2]

**Comment by cleebp** [comment]
https://www.kaggle.com/c/data-science-bowl-2017#tutorial

See section #isolation-of-the-lung-region-of-interest-to-narrow-our-nodule-search

**Comment by cleebp** [comment]
Working on ROI today in the branch `roi`, will hopefully have this all "done" by tonight and will merge the branch with master once ROI selection is complete.

**Comment by cleebp** [comment]
ROI selection is more or less finished, I ended up ditching the Kmeans method as I had better results using the original pre-processing tutorial's lung segment method to create the mask. As of now in the `driver` function after pre-processing the ROI mask is created and then I create a figure of the two images against each other (leaving just the lungs). From here we will have to perform feature extraction on the masked lungs themselves to then feed to SVM.

I also created a folder called `figures` in the root of our repo, the idea here is even though I'm not uploading all 20 patient's figures as we go through the steps, we can use the `figures` folder to selectively upload figures from major steps in our pipeline that way when we write the paper we will have relevant figures for each step for a single patient. Right now I have just placed Patient 1's figures in here for now. We should place a figure in there for patient 1 after feature extraction etc.

Finally I took out normalization and centering from the pre-processing step as it was messing with the ROI mask creation, we can probably add this back in after the mask is created but I'm not sure, I left the functions in the code but they aren't being called.
